Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -589,6 +589,26 @@ ClearFspHob (
   LdrGlobal->FspHobList = NULL;
 }
 
+
+/**
+  Set SPI flash EISS and LE and clear FSP HOBs.
+**/
+VOID
+SecurityCheck (
```

Comment:
Function name should be  ProgramSecuritySetting

Reply:
OK. let me change the function name.